frontend/document.js

```diff
@@ -5,17 +5,15 @@ import { renderToString } from 'react-dom/server';
 
 import assets from './lib/assets';
 import htmlTemplate from './htmlTemplate';
-
-import parseCAPI from './lib/parse-capi';
-import App from './App';
+import Article from './pages/Article';
 
 type Props = {
     data: {
         page: string,
         site: string,
-        body: { config?: {}, contentFields?: {} },
+        CAPI: CAPIType,
+        NAV: NavType,
```

Comment:
What is the significance of ALLCAPS? Props that will be passed to the Article?

Reply:
Honestly, I just preserved our naming convention here as we use that throughout and it's the same data.

It was possibly used to denote data that was got from unistore?

Reply:
Okay, no worries. Seems odd, but unrelated to your changes. We can fix it later if necessary.